src/calculateRentalCost.js

```diff
@@ -28,7 +28,18 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const price = 40;
+  const total = days * price;
+
+  if (days >= 7) {
+    return total - 50;
+  }
+
+  if (days >= 3) {
+    return total - 20;
+  }
```

Comment:
what a magic numbers 🪄 (days and amountOfDiscount)
lets create variables and then use them
it is better for managing them from one place and for understanding what do they mean